passive/CookieHTTPOnly.js

```diff
@@ -27,8 +27,13 @@ function scan(helper, msg, src) {
   var cookies = msg.getResponseHeader().getHeaders("Set-Cookie");
   if (cookies != null) {
     var re_noflag = /([Hh][Tt][Tt][Pp][Oo][Nn][Ll][Yy])/g;
-    if (!re_noflag.test(cookies)) {
-      helper.newAlert().setMessage(msg).setEvidence(cookies).raise();
+    if (!re_noflag.test(cookies.toString())) {
+      helper
+        .newAlert()
+        .setMessage(msg)
+        .setEvidence(cookies[0])
+        .setOtherInfo(`Other values: ${cookies.slice(1).toString()}`)
```

Comment:
Should we do this only if there are more? Would avoid showing `Other values: `, more a nit though.

Reply:
Yeah, I was also thinking that. Is it okay if I do that for all scripts in this PR?

Reply:
Sounds good.